axum/hello-world/README.md

```diff
@@ -0,0 +1,8 @@
+# Hello-World application using Actix
```

Comment:
The names are similar, but this is the readme for the axum hello world example, not actix :smile: